libs/portals/admin/ids-admin/src/components/forms/EditApplication/EditApplication.action.ts

```diff
@@ -34,6 +35,23 @@ const checkIfStringIsListOfUrls = (urls: string) => {
   return true
 }
 
+const checkIfStringIsArrayOfValidClaims = (claims: string) => {
+  if (claims.length === 0) {
+    return true
+  }
+
+  const array = splitStringOnCommaOrSpaceOrNewLine(claims)
+
+  const regex = /^\w+=\w+$|^\w+=\d+$/
```

Comment:
As `/w` is Alphanumeric characters it also contains `/d` so this regex can be simplified to 
```suggestion
  const regex = /^\w+=\w+$/
```